package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@typed-mxgraph/typed-mxgraph",
-  "version": "0.0.2-0",
+  "version": "0.0.4-0",
```

Comment:
In `mxgraph-type-definitiions`, we used to store the latest release version until a new release is done.
I don't know what we want do it for the `typed-mxgraph` package.
For sure, we cannot keep `0.0.2-0` as `0.0.3-0` is already available in npmjs

Reply:
cc @hungtcs